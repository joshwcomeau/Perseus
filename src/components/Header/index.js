// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import { css } from '../../helpers/global-aphrodite';

import styles from './styles';


const Header = () => {
  return (
    <div className={css(styles.header)}>
      Your Component Here :)
    </div>
  );
};

Header.propTypes = {

};

Header.defaultProps = {

};

export default Header;
