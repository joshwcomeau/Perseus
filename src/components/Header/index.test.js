/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
