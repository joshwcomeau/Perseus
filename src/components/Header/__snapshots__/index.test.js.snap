exports[`Header renders correctly 1`] = `
<div
  className="header_120drhm">
  Your Component Here :)
</div>
`;
